### ISSUE TITLE
Crawler docs seem broken. 

### ISSUE THREAD

**Issue by koaning** [deepset-ai/haystack-website#183]
I tried running the crawler code found on the [docs here](https://haystack.deepset.ai/components/preprocessing). However, it doesn't seem to run.

```python
from haystack.nodes import Crawler


crawler = Crawler(output_dir="crawled_files")
docs = crawler.crawl(
    urls=["https://haystack.deepset.ai/overview/get-started"],
    filter_urls=["haystack"],
    crawler_depth=1
)
```

But to no avail, there seems to be a dependency missing. 

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-10-ea7949ae73f6> in <module>
----> 1 from haystack.nodes import Crawler
      2 
      3 
      4 crawler = Crawler(output_dir="crawled_files")
      5 docs = crawler.crawl(

ModuleNotFoundError: No module named 'haystack.nodes'
​```

**Comment by koaning** [comment]
I found [this API page](https://haystack.deepset.ai/reference/crawler) that has another code-block. 

```
from haystack.connector import Crawler

crawler = Crawler(output_dir="crawled_files")
# crawl Haystack docs, i.e. all pages that include haystack.deepset.ai/overview/
docs = crawler.crawl(urls=["https://haystack.deepset.ai/overview/get-started"],
                     filter_urls= ["haystack\.deepset\.ai\/docs\/"])
```

But this code-block also seems to have issues. Is there a dependency missing in setup.py? 

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
~/Development/notebooks/venv/lib/python3.7/site-packages/haystack/connector/crawler.py in __init__(self, output_dir, urls, crawler_depth, filter_urls, overwrite_existing_files)
     45         try:
---> 46             from webdriver_manager.chrome import ChromeDriverManager
     47         except ImportError:

ModuleNotFoundError: No module named 'webdriver_manager'

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
<ipython-input-12-6878e96114dc> in <module>
      1 from haystack.connector import Crawler
      2 
----> 3 crawler = Crawler(output_dir="crawled_files")
      4 # crawl Haystack docs, i.e. all pages that include haystack.deepset.ai/overview/
      5 docs = crawler.crawl(urls=["https://haystack.deepset.ai/overview/get-started"],

~/Development/notebooks/venv/lib/python3.7/site-packages/haystack/connector/crawler.py in __init__(self, output_dir, urls, crawler_depth, filter_urls, overwrite_existing_files)
     46             from webdriver_manager.chrome import ChromeDriverManager
     47         except ImportError:
---> 48             raise ImportError("Can't find package `webdriver-manager` \n"
     49                               "You can install it via `pip install webdriver-manager`")
     50 

ImportError: Can't find package `webdriver-manager` 
You can install it via `pip install webdriver-manager`
```

**Comment by koaning** [comment]
Running pip manually did help though. 

```
pip install webdriver-manager selenium
```

**Comment by koaning** [comment]
The code now runs, but yields an empty result. 

```python
from haystack.connector import Crawler

crawler = Crawler(output_dir="crawled_files")
# crawl Haystack docs, i.e. all pages that include haystack.deepset.ai/overview/
docs = crawler.crawl(urls=["https://haystack.deepset.ai/overview/get-started"],
                     filter_urls= ["haystack\.deepset\.ai\/docs\/"])
```

This yields the following output. 

```
[WDM] - 

[WDM] - ====== WebDriver manager ======
[WDM] - Current google-chrome version is 94.0.4606
[WDM] - Get LATEST driver version for 94.0.4606
[WDM] - Driver [/Users/vincent/.wdm/drivers/chromedriver/mac64/94.0.4606.61/chromedriver] found in cache
```

The `docs` variable contains an empty list.

**Comment by tholor** [comment]
Hey @koaning,

Sorry, this issue slipped through and we only saw it now. Thanks for reporting! 
The issue with the above code is the ` filter_urls= ["haystack\.deepset\.ai\/docs\/"])` which basically enforces the crawled URLs to contain this str. As the site structure of our docs has changed there was no match and therefore an empty list of docs.

Something like this should work: 
```python
from haystack.nodes import Crawler
crawler = Crawler(output_dir="crawled_files")
docs = crawler.crawl(
    urls=["https://haystack.deepset.ai/overview/get-started"],
    filter_urls=["haystack"],
    crawler_depth=1
)
``` 

The code snippets have already been updated in both the API reference and the documentation (https://github.com/deepset-ai/haystack-website/commit/28137bc34546db734550629836522b53679547d3). So, I am closing this one now.

CC @brandenchan
